api/app/Http/Controllers/CollegeContoller.php

```diff
@@ -45,27 +36,20 @@ public function store(Request $request)
         return response()->json(['message' => 'Successfully created college!'], 200);
     }
 
-    /**
-     * Display the specified resource.
-     *
-     * @param  int  $id
-     * @return \Illuminate\Http\Response
-     */
     public function show($id)
     {
         //
-       $college = College::findOrFail($id);
+        $college = College::findOrFail($id);
+        $deans = RoleUser::with('user')->where('role_id', 4)->get();
```

Comment:
Create a scope function for the Dean
Ref: https://laravel.com/docs/9.x/eloquent#local-scopes
So that you will use `User::deans()`